contract_sale/security/ir.model.access.csv

```diff
@@ -2,6 +2,7 @@
 "contract_template_salesman","Recurring Salesman","contract.model_contract_template","sales_team.group_sale_salesman",1,1,1,0
 "contract_template_sale_manager","Recurring Sale Manager","contract.model_contract_template","sales_team.group_sale_manager",1,1,1,1
 "contract_line_saleman","Recurring Invoice Line Saleman","contract.model_contract_line","sales_team.group_sale_salesman",1,1,1,0
+"contract_model_modification","Contract Model Modification","contract.model_contract_modification","sales_team.group_sale_salesman",1,1,1,0
```

Comment:
```suggestion
"contract_modification_salesman","Contract Modification Salesman","contract.model_contract_modification","sales_team.group_sale_salesman",1,1,1,0
```

Sorry for the nitpicking, the rule is fine bu in the id and the name I think it is best to put the model name and the name of the group.

Reply:
Thanks! done :)